### PR TITLE
LPS-190001 Replace aui:button with clay:button

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
The motivation behind this PR stadirize the usages of buttons in the portal by using for clay:button or clay:icon.
https://liferay.atlassian.net/browse/LPS-190001
https://liferay.atlassian.net/browse/LPS-190002

Proposed Solution
========
The proposed solution consist in substitute the usages of aui:button for clay:button o clay:icon.

Steps to Verify
========
asset-categories button:
Product Menu -> Categorization -> Categories -> Create a Vocabulary -> Create a new Category -> buttons at the bottom

asset-list-button
Product Menu -> Collection -> Create a Dynamic Collection -> Select Web content and basic web content subtype -> Click on filter by field -> the select button there

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/category/details.jsp [hunk 1]
```diff
@@ -193,11 +193,26 @@ renderResponse.setTitle(title);
...
+					label="save"
+					type="submit"
+				/>
 
-				<aui:button disabled="<%= assetCategoriesDisplayContext.isSaveAndAddNewButtonDisabled() %>" onClick='<%= liferayPortletResponse.getNamespace() + "saveAndAddNew();" %>' value="save-and-add-a-new-one" />
+				<clay:button
+					disabled="<%= assetCategoriesDisplayContext.isSaveAndAddNewButtonDisabled() %>"
+					displayType="secondary"
+					label="save-and-add-a-new-one"
+					onClick='<%= liferayPortletResponse.getNamespace() + "saveAndAddNew();" %>'
```
Comment (p2kmgcl) [inline modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/category/details.jsp:206]: Can you create another task to move this global function to a JS file, please? :smile:

**Review by p2kmgcl — APPROVED** [review]
Nice! :smi

**Comment by p2kmgcl** [comment]
ci:forward

File: modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/category/details.jsp (hunk 1 above)
Comment (Stephy6) [reply modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/category/details.jsp:206]: sure!

**Comment by ealonso** [comment]
ci:test:stable

**Comment by Tim-Cao** [comment]
ci:forward:force

**Pull request closed without merging**